src/app/copy/block-copy/block-copy-config.ts

```diff
@@ -0,0 +1,27 @@
+import { CopyConfigBase } from '../copy-config-base';
```

Comment:
Considering the components are using inputs, this appears to be unused? 

We should remove the config classes and move the typedoc comments to each component input.

Reply:
So the thought here was to specify these shared values as part of a config class to ensure consistency in the names/types of component inputs. It was nice to be able to import this class in the tests and example files for both copy components as it sort of guided the development experience. I suppose the value here isn't has big as I first thought. I'll remove this config class and redefine these inputs for each of the copy components.

Reply:
That's an interesting idea and I did give that some thought. However, considering we have the same import properties on the component, it's just extra APIs to maintain. The fewer public APIs the better -- more difficult to deprecate later.

Going forward, I'd like to deprecate the config objects we have today in favor of component imports. It's easier for developers to bind properties directly to component tags rather than setup a config object in the view controller.

That said, we will likely keep some config objects for third party libraries. For example, using a config object to specify only C3 properties may still be helpful for chart components.

Reply:
Thanks for the clarification, that sounds like a great strategy.